api-doc.yml

```diff
@@ -185,6 +185,12 @@ paths:
           required: false
           schema:
             type: string
+        - in: query
+          name: maxLedgerVersionOffset
+          description: Offset from the current validated ledger version to the highest ledger version that the transaction can be included in. For example, for a transaction that will be valid for approximately 25-30 minutes, use a value of `500`.
```

Comment:
Optional: You may want to note the typical ledger validation time here rather than having folks infer it from 25-30 min and 500.